packages/cardano-services/src/ChainHistory/DbSyncChainHistory/queries.ts

```diff
@@ -173,13 +173,40 @@ export const findRedeemersByTxIds = `
 	WHERE tx.id = ANY($1)
 	ORDER BY redeemer.id ASC`;
 
+export const findVotingProceduresByTxIds = `
+	SELECT
+		tx.hash AS tx_id,
+		voter_role,
+		committee_voter,
+		dh.raw AS drep_voter,
```

Comment:
to avoid needing to do the bech32 formatting client side, can we use
```suggestion
		dh.view AS drep_voter,
```